tools/coverm/coverm_genome.xml

```diff
@@ -346,7 +346,7 @@
             $out.no_zeros
             -o output.tsv
             --threads \${GALAXY_SLOTS:-1}
-
+            2> stdout.txt
```

Comment:
Also, it would be nice to include an additional option for including the stdout.txt as output file.

Reply:
@gallardoalba thank you!